index.html

```diff
@@ -269,10 +268,7 @@ <h2>
         <div class="container">
           <div class="sectionsix__container__skills">
             <div class="sectionsix__container__skills__skill">
-              <h2>
-                About <br />
-                Myself
-              </h2>
+              <h2>About Myself</h2>
               <p>
                 Hello I’m a software developer! I can help you build a product ,
                 feature or website Look through some of my work and experience!
```

Comment:
[REQUIRED] The right-hand side should be taking more space than the left. About 40% for left and 60% for right

|Your Design|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/192989022-9cf4806c-9a6c-4ce2-99c7-96f11ef69c3c.png)|

[REQUIRED] Kindly align the languages' texts and icons to the left

|Your Design|
|--------------|
|![image](https://user-images.githubusercontent.com/62938735/192989820-e27fb3c5-7ef3-49cf-8dff-1a83d9babe35.png)|

Reply:
- Thanks for the review 😊I appreciate that I will do these changes, have a good day ❤️

Reply:
- Thanks @fobadara  🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊